tests/functional/spec/cookie_banner.spec.js

```diff
@@ -0,0 +1,32 @@
+import InitialPage from "../generated_pages/checkbox/mandatory-checkbox.page";
+
+describe('Given I open runner"', () => {
```

Comment:
Minor but for consistency, capitalise the keyword i.e `Given, When, Then`.

Reply:
Ok, added all the changes, thanks!